.changeset/pretty-places-visit.md

```diff
@@ -0,0 +1,6 @@
+---
+'@leafygreen-ui/drawer': minor
+---
+
+- Adds `inlineChildrenInTitle` prop to `Drawer` and `DrawerToolbarLayout` components. This prop allows for inline children to be rendered to the right of the title.
```

Comment:
Not crazy about `inlineChildrenInTitle`, but `inlineChildren` to me is not obvious that the inline children will be in the title. When Terrence is back, I'll see what prop name he's using for the toolbar

Reply:
Terrence is using `children` because the slot can technically accept anything, like icons or a select component. In this case, we just want icons. So we can align with `Code`, which uses `customActionButtons` or we can go with one of your suggestions. I like `customHeaderActions` and `inlineHeaderActions`. I can bring this up during dev sync so we're all aligned.

Reply:
Update: During dev sync, we agreed to remove this new prop and have only the `title` prop. If a string is passed to `title`, we will render the title with the correct styles and semantic tag. If a React node is passed, the consumer is responsible for the styles and adding the correct semantic tag. An recipe/example will be added to the README.